docs/.pages

```diff
@@ -2,7 +2,6 @@ nav:
   - Introduction: index.md
   - installation
   - tutorials
-  - how-to-guides
```

Comment:
Please confirm this comment and tell me your thoughts. :bow: 
https://github.com/autowarefoundation/autoware-documentation/pull/147#issuecomment-1161348501